### DISCUSSION TITLE
How do the purchasable licenses and open source interact?

### DISCUSSION THREAD

**Discussion by My1** [rustdesk/rustdesk#5931] [Q&A]
considering rustdesk is open source, couldnt one just use the server and mod out the paywall and licensing stuff?
not arguing against having paid options for open source software it sure needs money, but just the curious question how it all interacts with the open source.

**Comment by Darin755** [comment]
I personally will never use something that has the source code locked away. I night as well use proprietary commercial solutions at that point.
